openpype/plugins/publish/collect_frame_range_data.py

```diff
@@ -0,0 +1,85 @@
+import pyblish.api
+import clique
+from openpype.pipeline import publish
+from openpype.lib import BoolDef
+
+
+class CollectFrameRangeData(pyblish.api.InstancePlugin,
+                            publish.OpenPypePyblishPluginMixin):
+    """Collect Frame Range data.
+    """
+
+    order = pyblish.api.CollectorOrder + 0.491
+    label = "Collect Frame Range Data"
+    families = ["plate", "pointcache",
+                "vdbcache", "online",
+                "render"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        repres = instance.data.get("representations")
+        asset_data = None
```

Comment:
The name `asset_data` is confusing. Because for the case of the image sequences it's  not actually the asset data. Can we make it `collected_data` or `frame_data` or whatever so it's clear that it's not the data from the asset entity itself.